core/camel-core-languages/src/main/docs/modules/languages/pages/simple-language.adoc

```diff
@@ -96,6 +96,8 @@ converted body can be null.
 
 |bodyOneLine | String | Converts the body to a String and removes all line-breaks so the string is in one line.
 
+|prettyBody | String | Converts the body to a String and pretty print if JSON or XML.
```

Comment:
Converts the body to a String, and attempts to pretty print if JSon or XML, otherwise the body is returned as the String value.